<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DimredTab</class>
 <widget class="QMainWindow" name="DimredTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topBar" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="transformLabel">
         <property name="text">
          <string>Transformation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="transformSelect">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="ChartView" name="view">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Plot Tools</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCycleBackward"/>
   <addaction name="actionCycleForward"/>
   <addaction name="actionComputeDisplay"/>
   <addaction name="actionSavePlot"/>
  </widget>
  <action name="actionComputeDisplay">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Compute New</string>
   </property>
   <property name="toolTip">
    <string>Select a new display transform to compute</string>
   </property>
  </action>
  <action name="actionSavePlot">
   <property name="icon">
    <iconset theme="camera-photo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Save the plot to SVG or PNG file</string>
   </property>
   <property name="shortcut">
    <string>Print</string>
   </property>
  </action>
  <action name="actionCycleForward">
   <property name="icon">
    <iconset theme="arrow-right">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cycle Forward</string>
   </property>
   <property name="toolTip">
    <string>Cycle dimension forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionCycleBackward">
   <property name="icon">
    <iconset theme="arrow-left">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cycle Backward</string>
   </property>
   <property name="toolTip">
    <string>Cycle dimension backwards</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartView</class>
   <extends>QGraphicsView</extends>
   <header>scatterplot/chartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
